### ISSUE TITLE
🚀 - [Skeleton] Accessibilty enhancements (aria) for notifying screen-readers

### ISSUE THREAD

**Issue by ajitzero** [taiga-family/taiga-ui#8999]
### Description

**Enhancement for:** https://taiga-ui.dev/directives/skeleton

**Brief:**
We should consider using one or more of the following aria attributes to inform assistive technologies regarding the current state of skeleton loaders:
- `aria-busy`: Most relevant.
- `aria-hidden`: _Technically_ correct, but is likely not applicable. (Mentioning it here for posterity. I'm not an expert.)
- `aria-live`: Disrupts the user's focus on status change, either immediately (default) or when the user is idle (`polite`). This is good, unless there are multiple notifications for a group of skeleton loaders.

**Problems:**
- `aria-live` is important, but difficult to assign to a group of skeleton elements. We may need to introduce a `[tui-skeleton-group]` directive for the parent, which solves this issue but is also easy for developers to forget to use, even if it exists.
- Is it possible to apply it only to the parent element and not recursively to the children? I may be misunderstanding how the component works, so this may be a non-issue. `aria-atomic` may help solve the issue if this _is_ a problem.

**Comment by AndreiBelokopytov** [comment]
I believe that the mandatory use of ARIA attributes for the skeleton does not align with the goals of TaigaUI users for the following reasons:

* There are various ways to utilize the attributes `aria-busy` and `aria-live` for dynamic content. Each specific scenario should be validated with accessibility tools, such as VoiceOver, to meet the application’s requirements.

* For example, the feed pattern at https://www.w3.org/WAI/ARIA/apg/patterns/feed suggests using `aria-busy` on the content block during updates, temporarily hiding all its content from accessibility tools. Therefore, there is no need to use `aria-busy` or `aria-hidden` on the skeleton elements.

* If the skeleton is placed outside the block with the aria-busy attribute, as described in this blog at https://adrianroselli.com/2020/11/more-accessible-skeletons.html, then `aria-hidden` should be used for the skeleton. However, as mentioned earlier, we cannot know if this pattern will actually be implemented in an application using TaigaUI.

* The `aria-live` attribute can be utilized for hidden elements that are not related to the skeleton but notify users of the completion of loading. An example can be found here: https://www.sarasoueidan.com/blog/accessible-notifications-with-aria-live-regions-part-1/#aria-busy%3A-please-wait-until-the-changes-are-complete.

In conclusion, it can be said that in a component library like TaigaUI, it is difficult to anticipate the optimal scenario for implementing an accessible page with dynamic content loading. This task, along with accessibility testing, should fall under the responsibility of application developers.

**Comment by vladimirpotekhin** [comment]
Based on what is written above, we decided to leave the responsibility for attributes to the end user
